docs/_includes/sidebar.html

```diff
@@ -14,40 +13,39 @@
 
   <div class="version-picker">
 
-    {% if page.url == '' %} 
-      {% assign ver3url = page.url | append: '/en/v3.x/' %} 
-      {% assign ver2url = page.url | append: '/' %} 
-    {% elsif page.url == '/' %} 
-      {% assign ver3url = page.url | append: 'en/v3.x/' %} 
-      {% assign ver2url = page.url %} 
+    {% if page.url == '' %}
```

Comment:
Since the 3.x index.md file moved to top-level and the 2.x index.md file moved down, I inverted these checks.